kwrelease/event.go

```diff
@@ -38,6 +41,8 @@ func (e *Event) Init() error {
 	e.currentRelease = e.GetRelease(e.CurrentReleaseSecret.Name)
 	e.previousRelease = e.getPreviousRelease()
 
+	e.GetConfigDiffYAML()
```

Comment:
Why call this if you don't capture the return value

Reply:
Good catch. That's a mistake. I was just forcing it to run during development so the diff would be logged. I removed the log line but forgot to remove this.

I wasn't expecting anyone to review it so I just merged it to master but I will remove this now.

Reply:
I'm going to tackle the issue with sensitive values being leaked in a separate PR.